src/RtcDS3231.h

```diff
@@ -569,33 +569,36 @@ template<class T_WIRE_METHOD> class RtcDS3231
         return (DS3231AlarmFlag)alarmFlags;
     }
   
-    DS3231AlarmFlag LatchAlarmOneFlags()
+        DS3231AlarmFlag LatchAlarmOneFlag()
```

Comment:
You added an extra indentation here, please remove.

Reply:
Totally missed this, thanks!